spec/helpers/groups/form_helper_spec.rb

```diff
@@ -0,0 +1,170 @@
+describe GroupsFormHelper do
+  let(:group) { create(:group) }
+  let(:inputs) { nil }
+  let(:action) { nil }
+  
+  before do
+    @group = group 
+  end
+  
+  let(:valid_inputs) do
+    [
+      {
+        id: 'group_name',
+        type: 'text',
+        name: 'group[name]',
+        label: t('common.name'),
+        value: 'Test Group',
+        required: true,
+        dark: true
+      },
+      {
+        id: 'group_description',
+        type: 'textarea',
+        name: 'group[description]',
+        label: t('common.form.description'),
+        value: 'Group description',
+        required: true,
+        dark: true
+      }
+    ]
+  end
+
+  let(:valid_action) { '/groups' }
+  
+  describe '#new_group_props' do
+    subject { new_group_props }
+
+    context 'has no existing group' do
+      before do
+        @group = nil 
+      end
+  
+      it 'returns correct props' do
+        expect(subject).to eq(
+          inputs: [
+            {
+              id: 'group_name',
+              type: 'text',
+              name: 'group[name]',
+              label: t('common.name'),
+              value: nil,
+              required: true,
+              dark: true
+            },
+            {
+              id: 'group_description',
+              type: 'textarea',
+              name: 'group[description]',
+              label: t('common.form.description'),
+              value: nil,
+              required: true,
+              dark: true
+            },
+            { id: 'submit', type: 'submit', value: 'Submit', dark: true }
+          ],
+          action: '/groups'
+        )
+        
+      end
+    end
+
+    context 'has valid arguments' do
+      let(:inputs) { valid_inputs }
+      let(:action) { valid_action }
+      it 'returns correct props' do
+        expect(subject).to eq(
+          inputs: [
+            {
+              id: 'group_name',
+              type: 'text',
+              name: 'group[name]',
+              label: t('common.name'),
+              value: 'Test Group',
+              required: true,
+              dark: true
+            },
+            {
+              id: 'group_description',
+              type: 'textarea',
+              name: 'group[description]',
+              label: t('common.form.description'),
+              value: 'Group description',
+              required: true,
+              dark: true
+            },
+            { id: 'submit', type: 'submit', value: 'Submit', dark: true }
+          ],
+          action: '/groups'
+        )
+      end
+    end
+
+    describe '#edit_group_props' do
```

Comment:
I think the test is failing cause of how we nested these tests. If you look at the CircleCI failure you'll see the test reads as `#new_group_props #edit_group_props has invalid arguments returns correct props`. We need to move this describe block to level above.

Reply:
Ahh gotcha, corrected!  It looks like the CircleCi issue now is with the OmniAuth gem.  It's suggesting that it be disabled until patch is available.  Wane me to disable it?

Reply:
Oh :O We don't want to disable it though since it's needed for authentication.

How did you figure out it was an issue with this gem? Is there a page you can link me to?

Reply:
Oh, no worries!  I just saw it in the audit check if you scroll down in CircleCi: https://circleci.com/gh/ifmeorg/ifme/8697?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

Reply:
Ahh, that is a valid issue. I'll merge your tests since they pass! I'll take care of the gem vulnerability issue, thanks for calling it out!